File: PH2007SDK/developpers/VBTut1/Collector.vb
'Tutorial done by
' 
'Richard Clark
'Project Hoshimi Lead Manager
'Contact at ori@c2i.fr
' 
Imports PH.Common

<PH.Common.Characteristics(ContainerCapacity:=20, _
    CollectTransfertSpeed:=5, _
    Scan:=5, _
    MaxDamage:=0, _
    DefenseDistance:=0, _
    Constitution:=20)> _
  Public Class Collector
    Inherits PH.Common.NanoCollector
End Class

<PH.Common.Characteristics(ContainerCapacity:=0, _
    CollectTransfertSpeed:=5, _
    Scan:=5, _
    MaxDamage:=5, _
    DefenseDistance:=12, _
    Constitution:=20)> _
  Public Class Defender
    Inherits PH.Common.NanoCollector
End Class

File: PH2007SDK/developpers/CSTut1/Collector.cs
/* Tutorial done by
 * 
 * Richard Clark
 * Project Hoshimi Lead Manager
 * Contact at ori@c2i.fr
 * 
*/
using System;
using System.Collections.Generic;
using System.Text;
using System.Drawing;
using PH.Common;
using PH.Map;

namespace CSTut1
{
    [PH.Common.Characteristics(ContainerCapacity = 20,
     CollectTransfertSpeed = 5,
     Scan = 5,
     MaxDamage = 0,
     DefenseDistance = 0,
     Constitution = 20)]
    public class Collector : PH.Common.NanoCollector
    { }

    [PH.Common.Characteristics(ContainerCapacity = 0,
    CollectTransfertSpeed = 5,
    Scan = 5,
    MaxDamage = 5,
    DefenseDistance = 12,
    Constitution = 20)]
    public class Defender : PH.Common.NanoCollector
    { }

}
